### PR TITLE
Add subsection "Data privacy when using a work profile"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2900#data-privacy-when-using-a-work-profile

AB#47343393

### CODE REVIEW

File: microsoft-edge/privacy-whitepaper/index.md [hunk 1]
```diff
@@ -645,6 +645,18 @@ By default, while browsing **InPrivate**, Microsoft does not collect any informa
 Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.  You can stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing. Open **Start** > **Settings** > **Privacy** and turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
 
 
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization can see the following data:
+* The application URL.
+* The identity of the user in the profile.
+* The file name.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:654]: @apryzgoda - need more context for "the file name".  Which file name, of what?

File: microsoft-edge/privacy-whitepaper/index.md [hunk 2]
```diff
@@ -645,6 +645,18 @@ By default, while browsing **InPrivate**, Microsoft does not collect any informa
 Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.  You can stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing. Open **Start** > **Settings** > **Privacy** and turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
 
 
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization can see the following data:
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:651]: need more verbose lead-in sentence to establish context

File: microsoft-edge/privacy-whitepaper/index.md [hunk 3]
```diff
@@ -634,15 +634,39 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.
+
+To stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing:
+
+* On Windows, open **Start** > **Settings** > **Privacy**, and then turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).<!-- todo: * On macOS, do X. -->
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:657]: @apryzgoda - how do this on Mac?

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (AmitMishaeli) [reply microsoft-edge/privacy-whitepaper/index.md:663]: New phrasing: When working with the work profile, the organization **may** see the following data:

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (AmitMishaeli) [reply microsoft-edge/privacy-whitepaper/index.md:657]: The new clause is not relevant for InPrivate mode

**Comment by AmitMishaeli** [comment]
When working with the work profile, the organization may see the following data:
For example, when the user works in the profile of user1@contoso.com, Contoso may see the data.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:657]: i will remove hidden comment

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:663]: i'll revise

File: microsoft-edge/privacy-whitepaper/index.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:657]: removed comment, smoothed paragraph

File: microsoft-edge/privacy-whitepaper/index.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:663]: changed to 'may'

File: microsoft-edge/privacy-whitepaper/index.md [hunk 4]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.
+
+To stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing, open **Start** > **Settings** > **Privacy**, and then turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
+
+
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization may see the following data:
+* The application URL.
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:662]: What do you mean by application URL? Is this just any URL i go to on the web, basically saying they can see history? Or is this a specific type of URL?

File: microsoft-edge/privacy-whitepaper/index.md [hunk 5]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.
+
+To stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing, open **Start** > **Settings** > **Privacy**, and then turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
+
+
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When working with the work profile, the organization may see the following data:
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:661]:
```suggestion
When using a work profile, the organization may see the following data:
```

**Review by apryzgoda — CHANGES_REQUESTED** [review]
I've added some preliminary comments. More context needed for me to fully understand and review this.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (AmitMishaeli) [reply microsoft-edge/privacy-whitepaper/index.md:664]: 'The content of files': The content of files the user attempts to download or upload.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 4 above)
Comment (apryzgoda) [reply microsoft-edge/privacy-whitepaper/index.md:662]: @AmitMishaeli

File: microsoft-edge/privacy-whitepaper/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:664]: @AmitMishaeli - regarding specifically the list item "The file name", I think clarifying for context might give:

The file name of files that the user attempts to download or upload.

correct?  is there better wording?

File: microsoft-edge/privacy-whitepaper/index.md (hunk 5 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:661]: changed

File: microsoft-edge/privacy-whitepaper/index.md (hunk 4 above)
Comment (AmitMishaeli) [reply microsoft-edge/privacy-whitepaper/index.md:662]: Hi @apryzgoda ,
It is the URL used to go to the web.
We do not see the browsing history.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 6]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.
+
+To stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing, open **Start** > **Settings** > **Privacy**, and then turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
+
+
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When using a work profile, the organization may see the following data:
+* The application URL.
```
Comment (apryzgoda) [inline microsoft-edge/privacy-whitepaper/index.md:662]:
```suggestion
* The URL.
```

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

File: microsoft-edge/privacy-whitepaper/index.md [hunk 7]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+Browsing data for the specific **InPrivate** session is cleared after all **InPrivate** windows are closed.  When using the Windows Input Method Editor (IME) keyboard for typing and inking, data may be collected to improve language recognition and suggestion capabilities.
+
+To stop inking and typing data from being collected by Microsoft while using the Windows IME keyboard during **InPrivate** and normal browsing, open **Start** > **Settings** > **Privacy**, and then turn off **Inking & typing personalization**.  For more information about **InPrivate** browsing, see [Browse InPrivate in Microsoft Edge](https://support.microsoft.com/help/4533513).
+
+
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When using a work profile, the organization may see the following data:
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:661]:
```suggestion
When using a work profile, Microsoft may see the following data:
```

File: microsoft-edge/privacy-whitepaper/index.md [hunk 8]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+* The URL.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:662]:
```suggestion
* The URLs accessed by the user.
```
@AmitMishaeli - please review this suggested change.

File: microsoft-edge/privacy-whitepaper/index.md [hunk 9]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+* The identity of the user in the profile.
+* The file names of files that the user attempts to download or upload.
+* The content of files that the user attempts to download or upload.
+* Any attempted actions, such as cut, copy, print, download, or upload.
+* The operating system, browser, and browser version.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:667]:
```suggestion
* The user's operating system, browser, and browser version.
```
@AmitMishaeli - please review this suggested change.

rendered h2 section:
https://review.learn.microsoft.com/en-us/microsoft-edge/privacy-whitepaper/?branch=pr-en-us-2900#profile

File: microsoft-edge/privacy-whitepaper/index.md [hunk 10]
```diff
@@ -634,15 +634,37 @@ Profiles in Microsoft Edge allow you to separate your browsing data into indepen
...
+
+<!-- ------------------------------ -->
+#### Data privacy when using a work profile
+
+When using a work profile, the organization may see the following data:
+* The URL.
+* The identity of the user in the profile.
+* The file names of files that the user attempts to download or upload.
+* The content of files that the user attempts to download or upload.
+* Any attempted actions, such as cut, copy, print, download, or upload.
```
Comment (mikehoffms) [inline microsoft-edge/privacy-whitepaper/index.md:666]:
```suggestion
* Any attempted actions by the user, such as cut, copy, print, download, or upload.
```

File: microsoft-edge/privacy-whitepaper/index.md (hunk 7 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:650]: changed to:
When a user is signed in via a work profile, the user's organization has access to the following data.  For example, when a user is signed in with the profile user1@contoso.com, Contoso may see this data:

which incorp's [comment](https://github.com/MicrosoftDocs/edge-developer/pull/2900#issuecomment-1788771758) from main page of PR.

File: microsoft-edge/privacy-whitepaper/index.md (hunk 8 above)
Comment (mikehoffms) [reply microsoft-edge/privacy-whitepaper/index.md:652]: applied the change

**Review by AmitMishaeli — APPROVED** [review]

**Review by mikehoffms — COMMENTED** [review]
This PR LGTM.

**Review by apryzgoda — APPROVED** [review]
Edge Privacy Approved

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**